config/index.js

```diff
@@ -6,52 +6,22 @@
  * Individual functions from utils are still covered in scope of those file's tests.
  */
 
-const fs = require('fs');
-const rc = require('rc');
-
-const envVars = require('../lib/constants/enviroment');
 const logLevels = require('../lib/constants/logLevels');
 const timestamp = require('../lib/constants/timestamp');
 const {validate, validators} = require('../lib/validataion');
-const {invalidConfigObj, invalidUserConfig} = require('../lib/texts');
-const {ENV_VARS} = require('../lib/mappings');
+const {invalidConfigObj} = require('../lib/texts');
 const {pickNonNil} = require('../lib/utils/common');
-const SuitestError = require('../lib/utils/SuitestError');
 
 const sentryDsn = 'https://1f74b885d0c44549b57f307733d60351:dd736ff3ac994104ab6635da53d9be2e@sentry.io/288812';
 const DEFAULT_TIMEOUT = 2000;
 
-const configFields = [
-	{
-		name: 'logLevel',
-		type: 'string',
-	},
-	{
-		name: 'disallowCrashReports',
-		type: 'bool',
-	},
-	{
-		name: 'continueOnFatalError',
-		type: 'bool',
-	},
-	{
-		name: 'timestamp',
-		type: 'string',
-	},
-	{
-		name: 'defaultTimeout',
-		type: 'number',
-	},
-];
-
-const launcherFields = [
+const overridableFields = [
 	'tokenKey', 'tokenPassword', 'testPackId', 'concurrency', // launcher automated
 	'username', 'password', 'orgId', 'deviceId', 'appConfigId', 'inspect', 'inspectBrk', // launcher intaractive
-	'logDir', 'timestamp', 'configFile', // launcher common
+	'logLevel', 'logDir', 'timestamp', 'configFile', 'continueOnFatalError', 'disallowCrashReports', 'defaultTimeout', // launcher common
 ];
-const allFields = [...configFields.map(({name}) => name), ...launcherFields];
 
-const main = {
+const main = Object.freeze({
 	apiUrl: 'https://the.suite.st/api/public/v2',
```

Comment:
is it still v2 in docs it v3 https://suite.st/docs/suitest-network-api/api-reference/ ?

Reply:
I only know that it still works with v2, I missed it when we switched to v3, is it used in other projects already?

Reply:
Short answer: yes.
Long answer:
v2 is officially deprecated. No new features will be added to v2 and also will be removed in some time.